### ISSUE TITLE
elabctl script not working on RHEL 7.3 (Maipo) 

### ISSUE THREAD

**Issue by folf** [elabftw/elabctl#7]
After manually installing some of the dependencies, I seem to have been able to get an installation of elabftw up and running, but the setup with elabctl does not work.

I will be happy to try and troubleshoot this, so the automated setup script will work

**Comment by NicolasCARPi** [comment]
Hello,

Please tell me what is not working and what dependencies were missing. Unfortunately, this distribution is untestable for me because I don't have a subscription.

Have a look at `/var/log/elabftw.log`.

**Comment by folf** [comment]
Basically, running
`elabctl install`
only returns the ascii art defined in the ascii function and then it stops.
Nothing is written to /var/log/elabftw.log

I am sorry this is not much information, is there any obvious things I could change in the install script to track how far the install script comes before quitting?

**Comment by NicolasCARPi** [comment]
So when you run `elabctl install`, it'll run the `init` function, which runs `getDistrib` which tries to guess what is the distribution. As you can see [in the source](https://github.com/elabftw/elabctl/blob/master/elabctl.sh#L120-L125) there is a special step for centos or rhel. Can you try to run this step manually and see if you have any error. Also, is `wget` installed?

If all goes well, can you try and see which packages are missing from the system and/or repo:
dialog, git, zip, dig, wget.

**Comment by folf** [comment]
I just tried to split the RED HAT / CENTOS part of the getDistrib function into two parts, the RED HAT part then containing only the declaration of PACMAN=" yum -y install", and that version of the script works just fine, as far as I can tell.

So, I am guessing the extra code written in the original was
1. not necessary
2. somehow breaks the install on Red Hat, but not on CentOS (I am assuming you have tested on CentOS?)

Should I fork and edit the source, to make a pull request?

I have also tried to remove the python-pip and dialog packages from the system, before testing the elabFTW install script again, and this also worked.
